client/my-sites/comments/comment-list/index.jsx

```diff
@@ -379,11 +379,14 @@ export class CommentList extends Component {
 
 		return (
 			<div className="comment-list">
-				{ isJetpack &&
-					<QueryJetpackSiteCommentsTree page={ page } siteId={ siteId } status={ status } />
-				}
-				{ ! isJetpack &&
-					<QuerySiteCommentsTree siteId={ siteId } status={ status } />
+				{ isJetpack
+					? <QuerySiteCommentsList
+						number={ 100 }
+						offset={ ( page - 1 ) * COMMENTS_PER_PAGE }
+						siteId={ siteId }
+						status={ status }
+					/>
+					: <QuerySiteCommentsTree siteId={ siteId } status={ status } />
```

Comment:
sometimes when the formatting like this is bad I just ditch the ternary in favor of two separate conditionals; your choice

```js
{ isJetpack && (
	<QuerySiteCommentsList
		number={ 100 }
		…
	/>
) }
{ ! isJetpack && (
	<QuerySiteCommentsTree … />
) }
```